torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1052,6 +1052,19 @@ XLANativeFunctions::convolution_backward_overrideable(
                      : at::Tensor());
 }
 
+at::Tensor XLANativeFunctions::copy(const at::Tensor& self,
+                                    const at::Tensor& src, bool non_blocking) {
+  TORCH_LAZY_FN_COUNTER("xla::");
+  return _copy_from(src, self, non_blocking);
```

Comment:
I was doing the same thing on the other pull request: https://github.com/pytorch/xla/pull/4477. And then it fails some of the CI. I guess this won't be the right approach. Can you maybe exclude this one and then land the remaining changes?